src/landing-page/types.ts

```diff
@@ -0,0 +1,28 @@
+import { ViewerChannelSettings } from "../aics-image-viewer/shared/utils/viewerChannelSettings";
+
+// TBD what URL parameters to include here
+export type ViewerArgs = {
+  baseurl: string;
+  cellid: number;
+  cellPath: string;
+  fovPath: string;
+  fovDownloadHref: string;
+  cellDownloadHref: string;
+  viewerSettings: ViewerChannelSettings;
+};
```

Comment:
Based on your suggestion, in #207 I retyped a similar data structure (the one you may have based this type on?) as `AppProps` within an `Omit`. This is just a note - I think it would be perfectly fine to merge this as-is, given that (as you note in the comment above) shaking down the structure and handling of URL parameters is a broader issue that can be addressed separately.